### DISCUSSION TITLE
Why three different CI systems configured?

### DISCUSSION THREAD

**Discussion by michalfita** [gyscos/cursive#766] [Q&A] [answered]
I wanted to add building examples in CI, but they I've figured there are three different CI systems configured in this repository:
1. `.github/workflows/rust.yml`
2. `.travis.yml`
3. `appveyor.yml`

May I know why? And if not all are in use, why not remove redundant ones?

**Comment by gyscos** [comment]
Hi, and thanks for the question!

These mostly come from historical reason. Travis was used first, before github actions was a thing (or at least before I learned about it). Appveyor was added next to test windows, as travis didn't easily support it. Github actions has since replaced both.

**Reply by michalfita** [reply]
What about removing deprecated ones then?

**Reply by gyscos** [reply, marked as answer]
Yup, appveyor has been removed for a while, and I just removed the travis file now after reading your post. Thanks for noticing!

**Reply by michalfita** [reply]
Great, thanks!
